lib/utilities/concordia_constants.dart

```diff
@@ -34,7 +34,10 @@ const String FG_BUILDING_INITIAL = "FG";
 const double FG_BUILDING_LATITUDE = 45.494354;
 const double FG_BUILDING_LONGITUDE = -73.578428;
 
-var buildingCoords = [
+const LatLng sgwCoordinates = LatLng(45.495835, -73.578789);
+const LatLng loyolaCoordinates = LatLng(45.458257, -73.640450);
+
+var buildingCoordinates = [
```

Comment:
May be worthwhile to combine this variable with the map below it to avoid having redundant data. Probably cleaner.

Reply:
Constants reorganized!